### PR TITLE
deno 0.35.0

### PR DESCRIPTION
**Pull request by waldyrious**
Created with `brew bump-formula-pr`.

Full command used:
```
brew bump-formula-pr \
  --version=0.35.0 \
  --url=https://github.com/denoland/deno/releases/download/v0.35.0/deno_src.tar.gz \
  deno
```

Release notes: [deno@v0.35.0](https://github.com/denoland/deno/releases/tag/v0.35.0).

### CODE REVIEW

**Comment by Bo98** [comment]
@BrewTestBot test this please

File: Formula/deno.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class Deno < Formula
   desc "Command-line JavaScript / TypeScript engine"
   homepage "https://deno.land/"
-  url "https://github.com/denoland/deno/releases/download/v0.34.0/deno_src.tar.gz"
-  version "0.34.0"
-  sha256 "e6439e04b6df8db8d5192f98ee89c7d3ba9e966816fc7bf0d46cb52dc2e797aa"
+  url "https://github.com/denoland/deno/releases/download/v0.35.0/deno_src.tar.gz"
+  version "0.35.0"
```
Comment (Bo98) [inline Formula/deno.rb]: Homebrew is able to extract the version from the URL so this line isn't needed.

```
==> brew audit deno --online
==> FAILED
Error: 1 problem in 1 formula detected
deno:
  * Stable: version 0.35.0 is redundant with version scanned from URL
```

Comment (chrmoritz) [reply Formula/deno.rb]: 🎉 Great that the version is no longer detected as `src` after the recent changes in `brew`. ✨ 

Also you might want to coordinate this with #50881.

**Comment by iMichka** [comment]
I merged #50881, so you will have to rebase this PR.

**Comment by waldyrious** [comment]
> I merged #50881, so you will have to rebase this PR.

Done :)

**Comment by waldyrious** [comment]
> 🎉 Great that the version is no longer detected as `src` after the recent changes in brew. ✨

Just for future reference, that was done in https://github.com/Homebrew/brew/pull/7071, right @chrmoritz? It's been bothering me as well for a while :)

**Pull request closed without merging**